src/DvsaLogger/Listener/ApiResponse.php

```diff
@@ -32,6 +32,8 @@ class ApiResponse implements ListenerAggregateInterface, LoggerAwareInterface
 
     /**
      * @var Log
+     *
+     * @psalm-suppress PropertyNotSetInConstructor
```

Comment:
is this psalm suppress needed here if we have one for the class?

Reply:
@psalm-suppress can be used either at function or line level. I don't believe it works at the class level
https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#docblock-suppression